vm/devices/user_driver/src/buffer_manager.rs

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like our formatter doesn't quite catch _duplicate_ copyright directives. Please remove one :)

Reply:
Can never be too safe when it comes to IP.